### PR TITLE
nlp-fc.scm

### PR DESCRIPTION
**Pull request by yantrabuddhi**
Added scm file to parse a sentence and apply r2l scheme rules on it. needed to port all the r2l scm rules with some editing and found some un-implemented stuff. besides that the main logic is complete and main porting effort is done over things. function nlp-parse-fc sentence rules-file-path

### CODE REVIEW

File: opencog/nlp/relex2logic/IMPERATIVE.scm [hunk 1]
```diff
@@ -9,7 +9,8 @@
 				(VariableNode "$verb")
 				(TypeNode "WordInstanceNode")
 			)
-			(TypedvariableNode
+			;(TypedVariableNode
+      (TypedVariableLink
```
Comment (williampma) [inline opencog/nlp/relex2logic/IMPERATIVE.scm]: This edit is doing nothing.

**Comment by AmeBel** [comment]
please move the forward chaining examples to the atomspace repository

File: opencog/nlp/relex2logic/IMPERATIVE.scm [hunk 2]
```diff
@@ -33,14 +34,14 @@
...
 
-(InheritanceLink (stv 1 .99) (ConceptNode "IMPERATIVE-Rule") (ConceptNode "Rule"))
-
-(ReferenceLink (stv 1 .99) (ConceptNode "IMPERATIVE-Rule") IMPERATIVE)
-
+;(InheritanceLink (stv 1 .99) (ConceptNode "IMPERATIVE-Rule") (ConceptNode "Rule"))
+(InheritanceLink (stv 1 .99) (ConceptNode "imperative-Rule") (ConceptNode "Rule"))
+;(ReferenceLink (stv 1 .99) (ConceptNode "IMPERATIVE-Rule") IMPERATIVE)
+(ReferenceLink (stv 1 .99) (ConceptNode "imperative-Rule") IMPERATIVE)
 ; This is function is not needed. It is added so as not to break the existing
```
Comment (AmeBel) [inline opencog/nlp/relex2logic/IMPERATIVE.scm]: The inheritancelink and referencelink are not needed

**Comment by AmeBel** [comment]
a CMakeList.txt file similar to https://github.com/opencog/atomspace/blob/master/opencog/scm/CMakeLists.txt is needed.

File: opencog/nlp/relex2logic/loader/load-rules.scm [hunk 3]
```diff
@@ -0,0 +1,56 @@
+;;
```
Comment (AmeBel) [inline opencog/nlp/relex2logic/loader/load-rules.scm]: I think it would be better to define a module for loading.

**Comment by AmeBel** [comment]
This shoud be a scheme module, and unit tests added once it is compared to the java based implementation of r2l

**Pull request merged**